{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 6",
      "fix_state": "Not affected",
      "package_name": "mutt",
      "cpe": "cpe:/o:redhat:enterprise_linux:6"
    },
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Not affected",
      "package_name": "mutt",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Not affected",
      "package_name": "mutt",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Not affected",
      "package_name": "mutt",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    }
  ],
  "threat_severity": "Moderate",
  "public_date": "2021-05-05T00:00:00Z",
  "bugzilla": {
    "description": "neomutt: Out of bounds read in IMAP parser",
    "id": "1957451",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1957451"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "9.1",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:H",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-119",
  "statement": "This flaw doesn't affect the Mutt versions shipped with Red Hat Enterprise Linux 6, 7 and 8 as it depends on QRESYNC feature included on newer versions of Mutt than the ones distributed by Red Hat.",
  "acknowledgement": "",
  "name": "CVE-2021-32055",
  "document_distribution": "",
  "details": [
    "Mutt 1.11.0 through 2.0.x before 2.0.7 (and NeoMutt 2019-10-25 through 2021-05-04) has a $imap_qresync issue in which imap/util.c has an out-of-bounds read in situations where an IMAP sequence set ends with a comma. NOTE: the $imap_qresync setting for QRESYNC is not enabled by default."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2021-32055\nhttps://nvd.nist.gov/vuln/detail/CVE-2021-32055"
  ]
}